src/renderer/components/DataSourceItem.tsx

```diff
@@ -81,4 +68,35 @@ const DataSourceItem: React.FC<DataSourceItemProps> = ({dataSource, showDataSour
 }
 
 
-export default DataSourceItem;
\ No newline at end of file
+export default DataSourceItem;
+
+
+function dataSourceToTextIconProps(dataSource: DataSourceState) {
+    let ecvId;
+    let label;
+    if (dataSource.title) {
+        ecvId = dataSource.title.split(' ', 1)[0].toLowerCase();
+        label = dataSource.title.substr(0, 3).toUpperCase();
+    }
+    if (!ecvId || !ECV_META.ecvs[ecvId]) {
+        // This is a CCI-store specific hack
```

Comment:
It there is a hack for the CCI-Store, we should consider using one for the pilot store as well.

Reply:
> It there is a hack for the CCI-Store [sic]

If we preserve the `ecvId` in the Zarrs then this should work too!